src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url('../images/star.svg');
+    width: $stars-width;
+    height: $stars-height;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+  width: $stars-width;
+  height: $stars-height;
+}
```

Comment:
You can create a universal variant for stars modifier. (1,2,3,4,5 - try use possibilities of SCSS)